### ISSUE TITLE
0 trainable params

### ISSUE THREAD

**Issue by SayanDas74** [nachi-hebbar/Transfer-Learning-ResNet-Keras#3]
I'm getting 0 trainable params on running summary( ). All params are showing non-trainable. Ouput shape is showing ? and Param# is 0 (inbuilt) for flatten and dense rows. I'm training my own dataset uploaded to kaggle

**Comment by OjaswaniB** [comment]
use:

### Import Libraries
from tensorflow.keras.applications import ResNet50
from tensorflow.keras.layers import Flatten, Dense
from tensorflow.keras.models import Sequential

### Create the base model (all layers trainable)
pretrained_model = ResNet50(include_top=False,
                             input_shape=(180, 180, 3),
                             pooling='avg',
                             weights='imagenet',
                           classes=5)

### Create a Sequential model
resnet_model = Sequential()

### Add the outputs of the pretrained model as the first layer
resnet_model.add(pretrained_model)

### Set the input shape for Flatten based on the pretrained model's output
input_shape = pretrained_model.output_shape[1:]

resnet_model.add(Flatten(input_shape=input_shape))
resnet_model.add(Dense(512, activation='relu'))
resnet_model.add(Dense(5, activation='softmax'))

### Build the model (important step)
resnet_model.build(input_shape=(None, 180, 180, 3))  # Provide the expected input shape
